e2e/runner.go

```diff
@@ -69,13 +69,13 @@ func (r *Runner) Test(providerHost, consumerHost string) error {
 	log.Info().Msg("Running tests for env: " + r.testEnv)
 
 	err := r.compose("run", "go-runner",
-		"go", "test", "-v", "./e2e/...", "-args",
-		"--provider.tequilapi-host", providerHost,
-		"--provider.tequilapi-port=4050",
-		"--consumer.tequilapi-host", consumerHost,
-		"--consumer.tequilapi-port=4050",
-		"--geth.url=ws://ganache:8545",
-		"--consumer.services", r.services,
+		"/usr/bin/test", "-test.v",
+		"-provider.tequilapi-host", providerHost,
```

Comment:
should be no difference single vs double dashes. Postfix notation usually prefers double dashes.